src/LibraryManagementSystem.java

```
@@ -9,6 +9,33 @@ public class LibraryManagementSystem {
     private static int transactionQuantity = 0;
 
     public static void main(String[] args) {
-
+    	
+    }
+    
+    /*
+     * This method takes a string value as a parameter (title, author or ISBN)
+     * and lists the name, author, ISBN and page numbers of the book or books that match this string value.
+     */
+    public static void searchBooks (String string) {
+    	for (int i = 0; i < books.length; i++) {
+			if (books[i][0] == string) {
+				System.out.println("Tittle: " + books[i][0]);
+				System.out.println("Author: " + books[i][1]);
+				System.out.println("ISBN: " + books[i][2]);
+				System.out.println("Page Number: " + books[i][3]);
+			}
+			else if (books[i][1] == string) {
+				System.out.println("Tittle: " + books[i][0]);
+				System.out.println("Author: " + books[i][1]);
+				System.out.println("ISBN: " + books[i][2]);
+				System.out.println("Page Number: " + books[i][3]);
+			}
+			else if (books[i][2] == string) {
+				System.out.println("Tittle: " + books[i][0]);
+				System.out.println("Author: " + books[i][1]);
+				System.out.println("ISBN: " + books[i][2]);
+				System.out.println("Page Number: " + books[i][3]);
+			}
+		}
```

if the book is not found it should return an error message